robot_soccer_kit/robot.xml

```diff
@@ -0,0 +1,1741 @@
+<?xml version="1.0"?>
```

Comment:
Can you give this XML a more descriptive name? Preferable the name of the robot.